PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle

```diff
@@ -54,10 +54,11 @@ tasks.withType(JavaCompile) { options.encoding = 'UTF-8' }
 //}
 
 intellij {
+    type = 'IU'
```

Comment:
will this change make our plugin only available for Ultimate version?

Reply:
No.